src/components/copyToClipboard/index.tsx

```diff
@@ -24,6 +25,9 @@ interface ICopyToClipboardProps {
   showToast?: boolean;
   className?: string;
   iconSize?: number;
+  textDisplay?: string;
+  textDisplayProps?: HTMLAttributes<HTMLParagraphElement>;
+  textClassName?: string;
 }
 
 export const CopyToClipboard = memo((props: ICopyToClipboardProps) => {
```

Comment:
Looks to me that we are overcomplicating this component, lets think together how we could achieve this simpler

Reply:
Thanks for the suggestion! I simplified the component and achieved this by using the Tailwind peer utility